app/models/setting.rb

```diff
@@ -32,8 +32,17 @@ class Setting < ApplicationRecord
 
   validate :value_matches_type, if: Proc.new{ !self.read_attribute(:value).nil? }
   validate :parent_value_matches_parent_type, if: Proc.new{ self.parent_key.present? }
-  
-  # Needed to correctly write boolean true and false as value in specs  
+
+  def self.get_setting_by_key(key)
```

Comment:
These seems very similar to https://github.com/sparc-request/sparc-request/pull/1187/files which added `Setting.safe_value(key)`. Instead of creating two methods which are nearly identical except one returns nil and one returns an empty string, can we combine them somehow into a single method?

Reply:
I did some testing with this. Using string interpolation, nil actually converts to an empty string.
rails console:
```
"test #{nil}"
=> "test "
```
So we should be able to use the same `Setting.safe_value(key)` method defined in #1187